libobs/util/platform.c

```diff
@@ -559,7 +559,7 @@ static inline void from_locale(char *buffer)
 double os_strtod(const char *str)
 {
 	char buf[64];
-	snprintf(buf, 64, "%s", str);
+	snprintf(buf, sizeof(buf), "%s", str);
```

Comment:
Seems like weird use of `snprintf`, `strncpy` / `strncpy_s` (if available) seem more appropriate.

Reply:
Unfortunately `strncpy` is unsafe as-is as it doesn't null terminate on overflow. The preferred pattern is e.g. 
```
strncpy (buf, sizeof(buf)-1, str);
buf[sizeof(buf)-1] = 0;
```

Reply:
Right, will change.